api.planx.uk/send/createSendEvents.ts

```diff
@@ -25,17 +25,13 @@ const createSendEvents = async (req: Request, res: Response, next: NextFunction)
     }
 
     if ("uniform" in req.body) {
-      // Only send valid payloads to Uniform
-      // Fail silently - error already caught in frontend by Airbrake
-      if (req.body.uniform.body.xml) {
```

Comment:
Removing this condition is a leftover from moving OneApp generation out of the frontend.